app/model/Group.php

```
@@ -0,0 +1,238 @@
+        }
+    }
+
+    /**
+     * @return int
+     * @throws \Nette\Security\AuthenticationException when user is not logged in
+     */
+    private function getCurrentUserId()
+    {
+        $identity = $this->user->getIdentity();
+        if ($identity === null) {
+            throw new \Nette\Security\AuthenticationException('User is not logged in.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '234', column '23'). (undefined)

(MissingImport)